a.js

```diff
@@ -1,24 +1,22 @@
-let s=document.querySelector("#topBar_and_Video_container");
 
-let bt=document.querySelector("#three_bar");
 
-let s_b=document.querySelector("#search_button");
-let v_c=document.querySelectorAll("#video_container .vid_list");
 
 
+let s=document.querySelector(".topBar_and_Video_container");
+let bt=document.querySelector("#three_bar");
 bt.onclick = ()=>{
     
-    if(  s.id==="topBar_and_Video_container")
-        {
-            
-            s.id="large_topBar_and_Video_container";
-            
-        }
-        else 
-        {
-            s.id="topBar_and_Video_container";
-            
-        }
+      
+    if (s.classList.contains("large_topBar_and_Video_container")) {
+       
+        s.classList.remove("large_topBar_and_Video_container");
+        s.classList.add("topBar_and_Video_container");
+    } else {
+       
+        s.classList.remove("topBar_and_Video_container");
+        s.classList.add("large_topBar_and_Video_container");
+    }
```

Comment:
You can write this in a much better way :point_down: 
`let addClass = "topBar_and_Video_container"`
`let removeClass = "large_topBar_and_Video_container"`
` if (s.classList.contains("large_topBar_and_Video_container")) {
          addClass = "topBar_and_Video_container";
          removeClass = "large_topBar_and_Video_container";
}
s.class.add(addClass);
s.class.remove(removeClass);
`